workbench-core/environments/src/postDeployment/cognitoSetup.ts

```diff
@@ -17,9 +14,15 @@ export default class CognitoSetup {
     AWS_REGION: string;
     ROOT_USER_EMAIL: string;
     USER_POOL_NAME: string;
+    STACK_NAME: string;
   };
 
-  public constructor(constants: { AWS_REGION: string; ROOT_USER_EMAIL: string; USER_POOL_NAME: string }) {
+  public constructor(constants: {
+    AWS_REGION: string;
+    ROOT_USER_EMAIL: string;
+    USER_POOL_NAME: string;
+    STACK_NAME: string;
```

Comment:
can we just extract STACK_NAME (swb-stage-region) from USER_POOL_NAME (swb-userpool-stage-region), instead of passing as a param ?

Reply:
Good idea.